core-blocks/index.native.js

```diff
@@ -1,21 +1,4 @@
-/**
- * WordPress dependencies
- */
-import {
-	registerBlockType,
-} from '@wordpress/blocks';
-
 /**
  * Internal dependencies
  */
-import * as code from './code';
-import * as more from './more';
-
-export const registerCoreBlocks = () => {
-	[
-		code,
-		more,
-	].forEach( ( { name, settings } ) => {
-		registerBlockType( name, settings );
-	} );
-};
+import '../packages/core-blocks/src';
```

Comment:
We probably can remove this file and ensure that Mobile loads the package directly.

@hypest any thoughts?